packages/siwx/src/core/SIWXVerifier.ts

```diff
@@ -3,13 +3,9 @@ import type { SIWXSession } from '@reown/appkit-core'
 
 export abstract class SIWXVerifier {
   public abstract readonly chainNamespace: ChainNamespace
-  public abstract readonly messageVersion: string
```

Comment:
I'm removing this because it actually is not correct since the base verifiers are not validating the message content but the bytes generated by the signature in the string. When we create specific verifiers that checks the message content, then `shouldVerify` method should be overriden with the specific logic in the specific class.

Reply:
so if different message versions are used, verifiers are responsible to handle this change. 
aka if they change the message version, they should update their verifier?

Reply:
If the Verifier have a validation for something inside the message then yes, but the verifiers I have implemented so far only validates the resultant string signature, then it makes no sense to check for the version.